client/pages/post.jsx

```diff
@@ -139,12 +140,15 @@ export default class App extends React.Component {
     const isUploaded = !this.state.file
       ? <input className="absolute center-element" required type="file" name="image" ref={this.fileInputRef} accept=".png, .jpg, .jpeg" onChange={this.fileChangedHandler}/>
       : <input className="absolute center-element after-upload" required type="file" name="image" ref={this.fileInputRef} accept=".png, .jpg, .jpeg" onChange={this.fileChangedHandler}/>;
+    const imgPreview = this.state.img
+      ? <img className="image-preview" src={this.state.img}/>
+      : null;
     return (
       <div className="container">
         <form className="memory-form" onSubmit={this.handleSubmit}>
           <div className="image-upload">
                 { isUploaded }
-                <img className="image-preview" src={this.state.img}/>
+                { imgPreview }
```

Comment:
In JSX code, we use `&&` instead of a ternary that includes `null`.

https://reactjs.org/docs/conditional-rendering.html#inline-if-with-logical--operator